tests/vmi_lifecycle_test.go

```diff
@@ -1267,12 +1267,12 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				// Check if the stop event was logged
 				By("Checking that virt-handler logs VirtualMachineInstance deletion")
 				/*
-						Since we deleted the VMI object, there are two possible outcomes and both are expected:
-						1. virt-controller kicks in, registers a deletion request on the launcher pod and K8s deletes the pod
-					       before virt-handler had a chance to set or check the deletion timestamp on the domain.
-						2. virt-handler detects the deletion timestamp on the domain and removes it.
+						        Since we deleted the VMI object, there are two possible outcomes and both are expected:
```

Comment:
Accidental changes?

Reply:
yes, let me clean up them

Reply:
For some reason, My Visual studio code is formatting it this way when Iam saving the file. I am currently searching for a way to switch off the formatting.

Reply:
Done